scripts/invoke.py

```diff
@@ -128,13 +133,15 @@ def main_loop(gen, opt):
     doneAfterInFile = infile is not None
     path_filter = re.compile(r'[<>:"/\\|?*]')
     last_results = list()
+    if not os.path.isabs(opt.conf):
+        opt.conf = os.path.join(Globals.root,opt.conf)
```

Comment:
didn't this already happen above in line 67 and/or line 85?

Reply:
Yeah, it seems to be happening in three places. Got rid of the redundant ones.